### DISCUSSION TITLE
Issue with Audio Playback in Ant Media Server

### DISCUSSION THREAD

**Discussion by asifReapmind** [ant-media/Ant-Media-Server#6854] [Q&A]
I am experiencing an issue with audio playback on my Ant Media Server. Specifically, the audio is not playing when attempting to stream content. I would appreciate your assistance in resolving this matter.
 
 Here are the details of the issue: 
 Ant Media Server Version: 2.11.3 20240906_1141
  Streaming Protocol Used: hls Audio Codec: Audio: pcm_mulaw, 8000 Hz, mono, s16, 64 kb/s 
  
  Could you please assist me in identifying the cause of the issue and provide a solution? Any guidance or suggestions would be greatly appreciated.

**Comment by rahul78275** [comment]
Hello @asifReapmind 

Thank you for writing here. Can you please check the below points.

1. Check your device microphone is working or not.
2. Are you using community/enterprise application ? 
3. Have you tried testing the sample script ? (https://your_antmedia_url:5443/LiveApp/)
4. Are you using webrtc to publish content or any third party app like OBS (rtmp).

If possible you can share your antmedia url (not any credentials) to me at [me](mailto:rahul78275@gmail.com). We can debug together.
